config/recurring.yml

```
@@ -13,3 +13,7 @@ production:
   clear_solid_queue_finished_jobs:
     command: "SolidQueue::Job.clear_finished_in_batches(sleep_between_batches: 0.3)"
     schedule: every hour at minute 12
+  ready_for_cataloging:
+    class: ReadyForCatalogingJob
```

Could we get rid of a layer of indirection by changing this to:


```suggestion
    command: "SubmissionMailer.ready_for_cataloging.deliver_now"
```

That'll use ActionMailer's built-in job mechanics.

---

Oh, I was actually looking for that! I did not find the `command` option, that's great, thank you!

---

I could have looked at line 13... I need more coffee...